app/src/main/java/com/github/sdp/tarjetakuna/database/UserCardsViewModel.kt

```diff
@@ -60,15 +55,15 @@ class UserCardsViewModel : ViewModel() {
      * Remove the card from the user's collection
      */
     fun onRemoveButtonClick(card: MagicCard) {
-        usc.removeCardFromCollection(card)
+        usc.removeCardFromCollection(FBMagicCard(card, FBCardPossession.NONE))
```

Comment:
Why using a fake _FBCardPossession.NONE_ if you won't use it in _removeCardFromCollection_ ?

Reply:
yes, I just realised kotlin can have default arguments, which we should use, we keep every argument type as an FB card to keep the database code consistent.